types/src/on_chain_config/validator_set.rs

```diff
@@ -64,6 +67,22 @@ impl ValidatorSet {
     pub fn empty() -> Self {
         ValidatorSet::new(Vec::new())
     }
+
+    fn ordered_validators(payload: &[ValidatorInfo]) -> bool {
```

Comment:
Is there a function you can call directly (like Vector::sort) on payload here to sort and then compare the before and after?

Reply:
I mean, I guess we could, but why, when this works?

Reply:
Just nit I guess - I was wondering if this can be done in 2 lines instead of 13 lines.

Reply:
muahahaha, I like my silly approach... sort may have been better ... maybe?

fwiw, this is compiled away in release